src/pymmcore_widgets/_stage_widget.py

```diff
@@ -75,153 +190,141 @@ class StageWidget(QWidget):
 
+        self._poll_cb = QCheckBox("Poll")
         self.snap_checkbox = QCheckBox(text="Snap on Click")
-
         self._invert_x = QCheckBox(text="Invert X")
-        self._invert_y = QCheckBox(text="Invert Y")
-
-        self.radiobutton = QRadioButton(text="Set as Default")
-        self.radiobutton.toggled.connect(self._on_radiobutton_toggled)
-
-        top_row = QWidget()
-        top_row_layout = QHBoxLayout()
-        top_row_layout.setAlignment(AlignCenter)
-        top_row.setLayout(top_row_layout)
-        top_row.layout().addWidget(self.radiobutton)
-
-        bottom_row_1 = QWidget()
-        bottom_row_1.setLayout(QHBoxLayout())
-        bottom_row_1.layout().addWidget(self._readout)
-
-        bottom_row_2 = QWidget()
-        bottom_row_2_layout = QGridLayout()
-        bottom_row_2_layout.setSpacing(15)
-        bottom_row_2_layout.setContentsMargins(0, 0, 0, 0)
-        bottom_row_2_layout.setAlignment(AlignCenter)
-        bottom_row_2.setLayout(bottom_row_2_layout)
-        bottom_row_2.layout().addWidget(self.snap_checkbox, 0, 0)
-        bottom_row_2.layout().addWidget(self._poll_cb, 0, 1)
-        bottom_row_2.layout().addWidget(self._invert_x, 1, 0)
-        bottom_row_2.layout().addWidget(self._invert_y, 1, 1)
-
-        self.setLayout(QVBoxLayout())
-        self.layout().setSpacing(0)
-        self.layout().setContentsMargins(5, 5, 5, 5)
-        self.layout().addWidget(top_row)
-        self.layout().addWidget(self._btns, AlignCenter)
-        self.layout().addWidget(bottom_row_1)
-        self.layout().addWidget(bottom_row_2)
-
-        if self._dtype is not DeviceType.XYStage:
+        self._invert_y = QCheckBox(text=f"Invert {Y}")
+        self._set_as_default_btn = QRadioButton(text="Set as Default")
+        # no need to show the "set as default" button if there is only one device
+        if len(self._mmc.getLoadedDevicesOfType(self._dtype)) < 2:
+            self._set_as_default_btn.hide()
+
+        # LAYOUT ------------------------------------------------
+
+        # checkboxes below the move buttons
+        chxbox_grid = QGridLayout()
+        chxbox_grid.setSpacing(12)
+        chxbox_grid.setContentsMargins(0, 0, 0, 0)
+        chxbox_grid.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        chxbox_grid.addWidget(self.snap_checkbox, 0, 0)
+        chxbox_grid.addWidget(self._poll_cb, 0, 1)
+        chxbox_grid.addWidget(self._invert_x, 1, 0)
+        chxbox_grid.addWidget(self._invert_y, 1, 1)
+
+        main_layout = QVBoxLayout(self)
+        main_layout.setContentsMargins(5, 5, 5, 5)
+        main_layout.addWidget(self._set_as_default_btn, 0, Qt.AlignmentFlag.AlignCenter)
+        main_layout.addWidget(self._move_btns, Qt.AlignmentFlag.AlignCenter)
+        main_layout.addLayout(chxbox_grid)
+
+        # position label
+        # this can appear either below or to the right of the move buttons
+
+        if position_labels_below is None:
+            self._pos_labels_below = is_2axis
```

Comment:
I think this appears a bit strange to me, because if you have an XY stage and a Z stage side-by-side, the labels will appear differently despite passing the same arguments to the widget. i.e. given [this example](https://pymmcore-plus.github.io/pymmcore-widgets/widgets/StageWidget/#example) you'll have something like this:

![image](https://github.com/user-attachments/assets/4ad4b150-44c2-4094-8fe0-28229280ab17)

I'd prefer that if `position_labels_below` is `None`, then it should be `True`. Was there a reason you decided on this logic?

Reply:
i reverted that change.

Note that the example is also being [changed in this PR](https://github.com/pymmcore-plus/pymmcore-widgets/pull/334/files#diff-b3b7d3f2b3c3ba8a4f75f2be59d2bda2eaf7da47e63d27ecba02b133c9c2deecR20-R27), so had you followed the example after this PR (which encourages being explicit about your desired layout if you're using them in a group), it wouldn't look like that.  but it's fine with me to fallback to always having a vertical layout